### PR TITLE
JEP for Jenkins Twitter Team Policy

### PR DESCRIPTION
**Pull request by tracymiranda**
New policies for twitter team for Jenkins twitter account.

### CODE REVIEW

File: jep/0000/README.adoc [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+* Celebrate achievements of the community
+* Grow the open source community around Jenkins
+* Always be a trustworthy, reputable source of information
+
+==== Best Practices
+These are the best practices for contributors to follow:
+
+* Conduct - always abide by Jenkins community code of conduct .
+* Content - write or curate tweets that meet one of the goals above. 
+* Cadence - contributors should work together to maintain a regular cadence e.g. 3-5 tweets a day
```
Comment (daniel-beck) [inline jep/0000/README.adoc]: This seems like a _lot_. As in, tweeting pointless filler to reach a quota. Could we get 20-25 sample tweets for the last 7 day period, that would demonstrate that this is not unreasonably much?

File: jep/0000/README.adoc [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+== Security
+
+Contribution team members are recommended to use login verification on their twitter accounts.
+Contribution team members must report breaches to their Twitter account to the Twitter owner as soon as possible. 
+
+
+== Infrastructure Requirements
+
+A location to save Twitter monthly reports for sharing between team members would be required.
```
Comment (daniel-beck) [inline jep/0000/README.adoc]: Format?

File: jep/0000/README.adoc [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+
+== Backwards Compatibility
+
+There are no backwards compatibility concerns related to this proposal.
+
+
+== Security
+
+Contribution team members are recommended to use login verification on their twitter accounts.
```
Comment (daniel-beck) [inline jep/0000/README.adoc]: No experience with this, but shouldn't this just be a requirement, i.e. mandatory?

Comment (rtyler) [reply jep/0000/README.adoc]: There's no way to enforce this properly via TweetDeck

File: jep/0000/README.adoc (hunk 1 above)
Comment (rtyler) [reply jep/0000/README.adoc]: I think this sentence is easily fixed with "interesting/relevant tweets daily"

Comment (tracymiranda) [reply jep/0000/README.adoc]: Yes that works, will update in review pr

File: jep/0000/README.adoc (hunk 3 above)
Comment (tracymiranda) [reply jep/0000/README.adoc]: Will make this a requirement, but yes no way to enforce.

**Comment by tracymiranda** [comment]
@MarkEWaite thanks - yes will be great to get you involved!

File: jep/README.adoc [hunk 4]
```diff
@@ -52,6 +52,11 @@
 | link:https://github.com/kohsuke[Kohsuke{nbsp}Kawaguchi]
 | _not{nbsp}delegated_
 
+| Draft{nbsp}:speech_balloon:
```
Comment (oleg-nenashev) [inline jep/README.adoc]: it's autogenerated, right?

File: jep/10/README.adoc [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+=== Enabling a Team of Contributors
+
+Tweetdeck's Team feature will be used to manage the team of contributors.
+The following roles are recognized:
+
+* Owner - owner of the twitter account
+* Admin - can invite others to contribute
+* Contributor - creates content and can tweet, retweet and schedule tweets
+
+New contributors may be proposed at a governance meeting.
```
Comment (oleg-nenashev) [inline jep/10/README.adoc]: Everything at the governance meeting should be actually proposed in the mailing lists first. I would rather require an email to the Dev list

File: jep/10/README.adoc [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+
+== Security
+
+Contribution team members are recommended to use login verification on their twitter accounts.
+Contribution team members must report breaches to their Twitter account to the Twitter owner as soon as possible.
+
+
+== Infrastructure Requirements
+
+A location to save Twitter monthly reports for sharing between team members would be required.
```
Comment (oleg-nenashev) [inline jep/10/README.adoc]: Tweetdeck's configuration looks like a mandatory infra requirement.
Infra team should also have access to this account I'd guess

File: jep/10/README.adoc (hunk 5 above)
Comment (tracymiranda) [reply jep/10/README.adoc]: Good point - I'll update this to reflect that.

File: jep/10/README.adoc (hunk 6 above)
Comment (tracymiranda) [reply jep/10/README.adoc]: Maybe this could go on mailing list to see what folks think?

File: jep/README.adoc (hunk 4 above)
Comment (bitwiseman) [reply jep/README.adoc]: Yes? Why?

**Comment by DevMandy** [comment]
Is this open to the Jenkins community or just Cloudbees?  I'd love to be a jenkinsci Twitter contributor.

**Comment by MarkEWaite** [comment]
@DevMandy this is a community effort.  It is open to anyone willing to live by the rules and help the community.

**Pull request merged**